### DISCUSSION TITLE
Add permission to catch spawner by rank

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [PurpurMC/Purpur#456] [__Dismissed__] [answered]
Have a way to set a percentage to catch spawners by ranks, for example:
Silk Drop Percentage Rating:
   Legend = 50%
   Ultra 40%
   Super = 30%
   Default = 15%

**Comment by BillyGalbreath** [comment, marked as answer]
this is something best left up to a plugin. the server itself has no concept of ranks
